src/components/inputs/Dates/DateRange/translationTypes.ts

```diff
@@ -0,0 +1,56 @@
+export type Translation = {
```

Comment:
separated `translationTypes` and `translations` into their own files. i think these files should also probably be separate from `DateRange`, in a more centralized location.

Reply:
Hey @aleksarad! Thanks for updating this. I don't think you need this here, as you've already typed `translation` with `TranslatedStrings` [here](https://github.com/vimeo/iris/pull/99/files#diff-83742a70bca25fcb78f491bb5fe01842ef6ad3d148d7de56666a1001b4322321R61).

Reply:
oh yeah haha 🤔
ty, good catch! think this type being here is a leftover from the first iteration of this PR where the supported locales mattered.